bolt/bolt-protocol-message-specification-3.md

```diff
@@ -479,10 +481,10 @@ extra::Dictionary(
 
   For **Auto-commit Transaction** (`RUN`) the `extra` field carries:
 
-  - The `bookmarks` is a list of strings containg some kind of bookmark identification e.g ["neo4j-bookmark-transaction:1”, "neo4j-bookmark-transaction:2"]
-  - The `tx_timeout` is an integer in that specifies a transaction timeout in ms.
-  - The `tx_metadata` is a dictionary that can contain some metadata information, mainly used for logging.
-  - The `mode` specifies what kind of server the `RUN` message is targeting. For write access use `"w"` and for read access use `"r"`. Defaults to write access if no mode is sent.
+  - The `bookmarks` is a list of strings containing some kind of bookmark identification e.g `["neo4j-bookmark-transaction:1", "neo4j-bookmark-transaction:2"]`. Default if omitted: `[]`.
```

Comment:
nit: I'd probably just write `Default:` instead of `Default if omitted:` (this seems redundant)

Reply:
Could do. I felt like it'd be a good idea to explicitly hint readers that just not sending something is an option (and a good one too).

Reply:
hows about `Default:` with a note somewhere to explain?